app.css

```diff
@@ -69,4 +72,166 @@ body {
 .personal-social-info h3 {
   color: #7f8cff;
   font-size: 16px;
-}
\ No newline at end of file
+}
+
+/* section 2 */
+
+.portofolio {
+  display: grid;
+  gap: 20px;
+  margin-top: 100px;
+  height: auto;
+}
+.work-container {
+  padding: 10px;
+  background-color: white;
+  margin: 10px;
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.2);
+  border: 10px solid white;
+  border-radius: 10px;
+  height: auto;
+}
```

Comment:
[ Optional Change ] This change won't stop your project from being approved but will definitely make your project better and help you become a better developer.

- [ ] Kindly add bottom spacing between each work card.
Hint: You can add this style `margin-bottom: 88px;` as that is how it looks in the Figma design.

Your design
![image](https://user-images.githubusercontent.com/50941074/159119902-f8944628-fe62-46cc-99d5-026c759897e4.png)

The Figma Design
![image](https://user-images.githubusercontent.com/50941074/159119968-1e4fab52-e67f-49ac-a1ee-3ce4a706de19.png)